elastic/integrations
# Update ECS version 8.2.x to 8.5.1

- Enhancement

## What does this PR do?

Updates ECS version from 8.2.x to 8.5.1

## Checklist


## How to test this PR locally

Verify the ECS version updated in the listed integrations. Execute system and pipeline tests.

- Closes #4790
